LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini

```diff
@@ -136,37 +184,67 @@ MIDSHORT_CONVENTIONAL_RANGE[13] = -8
 MIDSHORT_CONVENTIONAL_RANGE[14] = -12
 MIDSHORT_CONVENTIONAL_RANGE[15] = -16
 MIDSHORT_CONVENTIONAL_RANGE[16] = -20
+
+; MEDLONG WEAPONS: MARKSMANRIFLE
+MEDLONG_CONVENTIONAL_RANGE[0] = 0
+MEDLONG_CONVENTIONAL_RANGE[1] = -15
+MEDLONG_CONVENTIONAL_RANGE[2] = -13
+MEDLONG_CONVENTIONAL_RANGE[3] = -12
+MEDLONG_CONVENTIONAL_RANGE[4] = -10
+MEDLONG_CONVENTIONAL_RANGE[5] = -9
+MEDLONG_CONVENTIONAL_RANGE[6] = -6
+MEDLONG_CONVENTIONAL_RANGE[7] = -3
+MEDLONG_CONVENTIONAL_RANGE[8] = 0
+MEDLONG_CONVENTIONAL_RANGE[9] = 0
+MEDLONG_CONVENTIONAL_RANGE[10] = 0
+MEDLONG_CONVENTIONAL_RANGE[11] = 0
+MEDLONG_CONVENTIONAL_RANGE[12] = 0
+MEDLONG_CONVENTIONAL_RANGE[13] = 0
+MEDLONG_CONVENTIONAL_RANGE[14] = 0
+MEDLONG_CONVENTIONAL_RANGE[15] = 0
+MEDLONG_CONVENTIONAL_RANGE[16] = 0
 ; TODO
 
 [LWOTC_WeaponPack.X2Item_LaserWeapons]
 ; ***** ImagePaths *****
 AssaultRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserRifle_Base"
+BattleRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserRifle_Base"
```

Comment:
Did realitymachina's pack not have these assets? If not I'll need to put in an art request. Let me know

Reply:
Correct. RM's pack reused both the 2D assets for images of the BR and MR as well as the based model for 3D assets. The only difference in 3D models is that the BR and MR use some different weapon parts (the MRs [except LS and CG] have scopes, for example). LS and CG MRs look identical to LS and CG assault rifles right now.